pytest_rpc/helpers.py

```diff
@@ -129,12 +145,8 @@ def delete_volume(volume_name, run_on_host):
     Raises:
         AssertionError: If operation unsuccessful.
     """
-    volume_id = get_id_by_name('volume', volume_name, run_on_host)
-    cmd = "{} openstack volume delete --purge {}'".format(utility_container,
-                                                          volume_id)
-    run_on_host.run_expect([0], cmd)
 
-    assert (asset_not_in_the_list('volume', volume_name, run_on_host))
+    delete_it('volume', volume_name, run_on_host)
```

Comment:
Should `def deleted` to be modified so it can take optional flags? For example on deleting volume, sometime we need to do `openstack volume delete --force volume_name` to delete the volume regardless of state or `openstack volume delete --purge` to remove associated snapshot.

Reply:
@phongdly I have updated the method internally to add these flags when the resource_type is defined as 'volume'.

Reply:
oops, looks like those flags cannot be used at the same time. I will implement a options parameter.